src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -276,16 +284,19 @@ def remove_group(fs, vol_spec, groupname):
 
 
 @contextmanager
-def open_group(fs, vol_spec, groupname):
+def open_group(fs, vol_spec, groupname, op_type=None):
```

Comment:
`op_type` isn't even passed to the Group class right now since it wasn't a requirement. What if you pass `None` as groupname when reading from .meta. Would `op_type` would still be necessary?

Reply:
We can do so, and that would be easier but there is this code inside `Group.__init__()` [here](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/volumes/fs/operations/group.py#L24) which gave me the impression operations are restricted for internal group -

```
            raise VolumeException(-errno.EPERM, "Operation not permitted for group '{0}' as it is an internal group.".format(groupname))
```

Therefore, will it be okay to do so?

Reply:
> It gave me the impression that operations on the internal group are restricted, therefore, will it be okay to do so?

I think so, yes - it should be possible without passing `op_type` for the operation you are trying to perform.

Reply:
Okay, will make this change.

Reply:
Done.